imap_processing/tests/codice/test_codice_l1a.py

```diff
@@ -40,5 +57,32 @@ def test_codice_l1a(test_file: Path, expected_filename: str):
     """
 
     packets = decom_packets(test_file)
-    cdf_filename = process_codice_l1a(packets)
+    _, cdf_filename = process_codice_l1a(packets)
     assert cdf_filename.name == expected_filename
+
+
+@pytest.mark.parametrize(("test_file", "validation_file"), TEST_VALIDATION_DATA)
+def test_l1a_validation_data(test_file: Path, validation_file: Path):
+    """Tests that the generated L1a CDF contents are valid.
+
+    TODO: This tests currently uses validation files that the pipeline itself
+    generates, rather than a CDF file generated by the CoDICE instrument team.
+    Once proper validation files are acquired, this test function should point
+    to those. This function currently just serves as a framework for validating
+    files, but does not actually validate them.
+    """
+
+    # Retrieve the validation file contents
+    validation_data = cdflib.xarray.cdf_to_xarray(validation_file)
```

Comment:
We have a `write_cdf()` helper, this got me to thinking that maybe we should also have a `load_cdf()` helper that basically just calls this function from `imap_processing` instead of needing to remember the nested imports of cdflib? Allowing us to leave `cdflib` as an implementation detail to our end users.

Reply:
I like the idea! I can take note of this and add that in a separate PR for the team to review/discuss.

Reply:
Yes, definitely a separate PR, thanks for volunteering! ;) Just wanted to write that down while I noticed it here :)